docs/build/dapp-staking/dapp_staking_architecture.md

```diff
@@ -0,0 +1,672 @@
+---
+sidebar_position: 1
+title: V3 Technical Architecture
+---
+
+## Introduction
+
+:::important
+Please note that this is a technical document on the design of the dApp Staking V3 as it has been imagined. It serves as a reference and helps to better understand its architecture. 
+
+:::
+
+For the final implementation on Astar Network or Shiden, please refer to the documentations on Astar Github.
+
+**Resources:**   
+https://docs.astar.network/docs/learn/dapp-staking/dapp-staking-protocol
+https://github.com/AstarNetwork/Astar/tree/master/pallets/dapp-staking-v3
+https://github.com/AstarNetwork/Astar/tree/master/precompiles/dapp-staking-v3
+
+# I. Overview
+
+### Eras & Periods
+
+The concept of eras is kept from the old dApp staking and a new concept of `periods` is introduced.
+
+Each `period` consists of two `subperiods` - `voting subperiod` followed by a `build&earn subperiod`:
```

Comment:
`Voting` and `Build&Earn` subperiod, to stay consistent with the rest of the docs.